daffodil-test/src/test/resources/org/apache/daffodil/usertests/MultipartBody.dfdl.xsd

```diff
@@ -54,7 +54,7 @@
 				<xsd:element name="BodyPart" dfdl:lengthKind="delimited"
 					minOccurs="1" maxOccurs="unbounded" type="xsd:string"
 					dfdl:occursCountKind="implicit" 
-          dfdlx:emptyElementParsePolicy="treatAsEmpty" />
+                    dfdl:emptyElementParsePolicy="treatAsEmpty" />
```

Comment:
Indentation in this file is totally goofed up. Rather than fix it, and have whitespace fixing be part of this PR, let's accumulate whitespace fixups on a ticket. 

I already added this file to this ticket: https://issues.apache.org/jira/browse/DAFFODIL-2759

Reply:
I went ahead and updated this from an earlier request.